src/Button.php

```diff
@@ -25,6 +27,18 @@ public function renderView()
             $this->addClass('icon');
         }
 
+        if ($this->rightIcon && !is_object($this->rightIcon)) {
+            $this->rightIcon = $this->add(new Icon($this->rightIcon), 'Content');
+
+            if ($this->content) {
```

Comment:
what is $this->icon or $this->rightIcon is already an object? Shouldn't we execute this part of code in such case too? (add class labeled, remove content etc.)